src/Form/Field/BaseField.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace Kirby\Form\Field;
+
+use Kirby\Form\Fields;
+use Kirby\Form\Mixin;
+use Kirby\Reflection\Constructor;
+use Kirby\Toolkit\HasI18n;
+
+/**
+ * Base class for any field type
+ *
+ * @package   Kirby Field
+ * @author    Bastian Allgeier <bastian@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ * @since     6.0.0
+ */
+abstract class BaseField
+{
+	use HasI18n;
+	use Mixin\Api;
+	use Mixin\DefaultValue;
```

Comment:
Why does the BaseField have a default value if we want to split next between InputField and DisplayField (the latter without any value)?